src/components/SearchResultItem.tsx

```diff
@@ -13,7 +13,6 @@ interface SearchResultItemProps {
     libraryContainer: LibraryContainer,
     highlightedText: string;
     detailed: boolean;
-    showItemSummary: boolean;
```

Comment:
@Enzo707 Is this property not used?

Reply:
@QilongTang I haven't seen any case of use